client/src/styles/form-group--no-divider.scss

```diff
@@ -0,0 +1,8 @@
+.form-group--no-divider {
```

Comment:
This change feels kind of hacky. Is this just ensuring the tick box isn't separated from the password fields? Can we add this to the existing password fields somehow? At least can we make the name of the file a little more semantic - perhaps like `cms-overrides.scss` or just `form-group.scss`? Having a scss file named after the only (specific) CSS class it defines seems... odd.

Reply:
No. The ConfirmedPasswordField returns a string of HTML, it is not possible to add the checkbox in the correct place without much use of even greater hacky code.
I could spend a whole bunch of time creating a whole new form field and getting it to render appropriately, then use `Injector` to replace all `ConfirmPassword` fields globally in the system...

Or rename the scss file.

Reply:
Haha sheesh okay. Agreed. Renaming the CSS file would be good.

Reply:
Updated name :)